laborator/content/stiva/sol/4-local-var/merge-arrays.asm

```diff
@@ -0,0 +1,114 @@
+%include "../utils/printf32.asm"
+
+section .data
+
+%define ARRAY_1_LEN 5
+%define ARRAY_2_LEN 7
+%define ARRAY_OUTPUT_LEN 12
+
+section .data
+array_1 db 27, 46, 55, 83, 84
+array_2 db 1, 4, 21, 26, 59, 92, 105
```

Comment:
```suggestion
array_1 dd 27, 46, 55, 83, 84
array_2 dd 1, 4, 21, 26, 59, 92, 105
```
Using `db` allows students to just `sub ARRAY_*_LEN` to allocate them onto the stack, thus hiding the fact that the logic should be `sub (arr_len * elem_size)`. This change makes students think about this.

Modify the rest of the code in the solution and in the skeleton accordingly.